app/dependencies.php

```diff
@@ -500,6 +511,23 @@ static function (ContainerInterface $c): Matching\Adapter {
             return new RoutableMessageBus($busContainer, $bus);
         },
 
+        SalesforcePushTransport::class => static function (): TransportInterface {
+            $transportFactory = new TransportFactory([
+                new AmazonSqsTransportFactory(),
+                new InMemoryTransportFactory(), // For unit tests (without Docker Compose setup).
```

Comment:
```suggestion
                new InMemoryTransportFactory(), // For unit tests.
```
(Docker Compose was only relevant before I put the `force`d vars in phpunit config)